servers/partner-ci/10.0/ui-fake.yaml

```diff
@@ -0,0 +1,234 @@
+---
+- parameter:
+    name: specific-parameters
+    parameters:
+      - text:
+          name: SPECIFIC_PROPS
+          default: |
+            UBUNTU_DIST=xenial;
+
+- project:
+    name: 'predefined_parameters'
+    fuel_release: '10.0'
+    fuel_release_checker: '10'
+    job_name: 'ui-fake'
+    nightly-name: '{fuel_release}.{job_name}.nightly'
+    custom_name: '{fuel_release}.{job_name}.custom'
+    reporter-name: '{fuel_release}.{job_name}.reporter'
+    nightly_timer: 'H 21 * * *'
+    nightly_disabled: false
+    force_erase: false
+    force_reuse: true
+    recreate_venv: false
+    debug: true
+    publish_results: true
+    fuel_ui_refs: ''
+    fuel_web_refs: ''
+    fuel_web_branch: 'origin/stable/newton'
+    fuel_ui_branch: 'origin/stable/newton'
+    docker_branch: 'fake-ui'
+    email_to: 'rkhozinov@mirantis.com'
+    scm_submodule_disabled: true
+    scm_wipe_workspace: false
+    skip_vote_successfull: false
+    skip_vote_failed: false
+
+    testrail_project: "Mirantis OpenStack"
+    testrail_suite: "[10.0][Fuel] UI testing"
+    testrail_suite_desc: "Suite description"
+    testrail_milestone: "10.0"
+    testrail_plan: "${{TREP_TESTRAIL_MILESTONE}} iso #${{SNAPSHOTS_ID}}"
```

Comment:
Change `iso #${{SNAPSHOTS_ID}}` to "ui" with timestamp, e.g. `ui 03/10/2017 18:30`